cargo-shuttle/src/lib.rs

```diff
@@ -879,12 +876,6 @@ impl Shuttle {
                 .context("archive entry")?;
         }
 
-        // Make sure to add any `Secrets.toml` files in the root of the workspace.
-        let secrets_path = self.ctx.working_directory().join("Secrets.toml");
-        if secrets_path.exists() {
-            tar.append_path_with_name(secrets_path, Path::new("shuttle").join("Secrets.toml"))?;
-        }
```

Comment:
This might be needed for non-workspace projects

Reply:
Yes, I think so, the problem is it is causing duplicate entries: https://app.circleci.com/pipelines/github/shuttle-hq/shuttle/2287/workflows/ef63bbba-4f8f-4f8e-9871-df8cafb33b63/jobs/41573 :thinking: